3rd-party/vaultpress.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+function jetpack_vaultpress_rewind_enabled_notice() {
+	$plugin_file = 'vaultpress/vaultpress.php';
+
+	deactivate_plugins( $plugin_file );
+	?>
+	<div class="notice notice-success">
+		<h2 style="margin-bottom: 0.25em;"><?php _e( 'Jetpack is now handling your backups.', 'jetpack' ); ?></h2>
```

Comment:
The headline "Jetpack is now handling your backups" is not necessarily true if Rewind is available but inactive. 

It sounds like (in the other thread) there is consensus that we want VaultPress disabled if Rewind is available, even if isn't active, so I don't want to tell users we're handling their backups if it's possible we're not. :)

Reply:
This message should only show if Rewind is actually configured, but we will confirm that in testing leading up to our wide launch.